.travis.yml

```diff
@@ -1,25 +1,24 @@
 dist: bionic
 language: go
 go:
-  - 1.12.x
+  - stable
   - 1.13.x
-  - 1.x
   - tip
 
 matrix:
   include:
-    - go: 1.x
+    - go: stable
       name: "verify-dependencies"
       script:
         - make verify-dependencies
-    - go: 1.12.x
+    - go: stable
```

Comment:
stable -> 1.13.x ?

Reply:
No, currently `stable` means 1.14.x. I didn't want to use an explicit version here since it will need to be changed every once in a while, but an implicit version like `stable` can stay as is.

Do you want it to be explicit?

Reply:
We aren't ready for 1.14 yet

Reply:
this is CI only and it passes.

OK I'll switch to explicitly 1.13

Reply:
updated